portfolio/src/main/webapp/script.js

```diff
@@ -13,21 +13,89 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * A wrapper around slowFill(), with the specific purpose of slowly
+ * filling out the biography section of the website.
  */
-function addRandomFact() {
-  const facts =
-      [
-        'I am 19 years old!',
-        'I once ran arch linux on Windows using WSL for the lolz!',
-        'My dogs\' names are Polly and Rex!',
-        'My \'to-read\' list is currently growing at a rate of about 2bpm (books per month)!',
-      ];
+const slowFillBiography = async () => {
+  const biography_text = 
+      'Hi! My name\'s Thomas, and I\'m just about to start my third year of ' +
+      'studies in University College Dublin. I\'m currently a STEP intern ' +
+      'at Google, and this portfolio aims to showcase my contributions ' + 
+      'during my time here.';
```

Comment:
I believe you can create multiline string just by using backtick (`). Just FYI.

Reply:
Yeah I saw that in the style doc, but the instructions also said that multi-line string literals could be implemented using just concatenation (https://google.github.io/styleguide/jsguide.html#features-strings-no-line-continuations). There was no clear indication as to which was preferable, so I wasn't sure which to use. I'll change this now though 👍